src/_layouts/default.html

```diff
@@ -9,6 +9,7 @@
     <meta property="og:image" content="http://www.decodemtl.com/assets/images/decodemtl-favicon-large.png">
     <link rel="stylesheet" href="/assets/css/app.css">
     <link href='http://fonts.googleapis.com/css?family=Lato:100,300,400,900' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
```

Comment:
Can you please change the two `<link>`s to be protocol-relative URLs (start with `//` instead of `http`)